### ISSUE TITLE
[Bug]: TestUnlockWithBindNotFound failed

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#11105]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

[logs_162277.zip](https://github.com/matrixorigin/matrixone/files/12291620/logs_162277.zip)

<img width="1418" alt="image" src="https://github.com/matrixorigin/matrixone/assets/77312370/bc14d1a5-bc58-4a79-96cf-35da365c692f">

https://github.com/matrixorigin/matrixone/actions/runs/5797095016/job/15711966076?pr=11102

**Comment by zhangxu19830126** [comment]
dup this #10962

**Comment by zhangxu19830126** [comment]
fixed by #11148
